### PR TITLE
finisi-static-pages

### PR DESCRIPTION
**Pull request by piiknight**

### CODE REVIEW

**Comment by piiknight** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    </div>
+    <div class="banner-w3text">
+      <div class="container">
+        <div class="w3l-agile">
+          <h2><%= t("layouts.header.stats").html_safe %></h2>
+          <div class="banner-pw3l">
+            <p><%= t("layouts.header.content").html_safe %></p>
+          </div>
+          <div class="w3lsmore">
+            <%= link_to( t("layouts.header.read"), about_path, :class =>"button-pipaluk button--inverted") %>
```
Comment (tranducquoc) [inline app/views/layouts/_header.html.erb]: class: "button-pipaluk button--inverted"

**Comment by tranducquoc** [comment]
commented

**Comment by piiknight** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**